docs/extensibility/checkout-flow-and-events.md

```diff
@@ -4,21 +4,21 @@ This document gives an overview of the flow for the checkout in the WooCommerce
 
 ## Table of Contents <!-- omit in toc -->
 
-- [General Concepts](#general-concepts)
-  - [Tracking flow through status](#tracking-flow-through-status)
-    - [`CheckoutProvider` Exposed Statuses](#checkoutprovider-exposed-statuses)
-      - [Special States:](#special-states)
-    - [`ShippingProvider` Exposed Statuses](#shippingprovider-exposed-statuses)
-    - [`PaymentMethodDataProvider` Exposed Statuses](#paymentmethoddataprovider-exposed-statuses)
-  - [Emitting Events](#emitting-events)
-    - [`onCheckoutValidationBeforeProcessing`](#oncheckoutvalidationbeforeprocessing)
-    - [`onPaymentProcessing`](#onpaymentprocessing)
-    - [`onCheckoutAfterProcessingWithSuccess`](#oncheckoutafterprocessingwithsuccess)
-    - [`onCheckoutAfterProcessingWithError`](#oncheckoutafterprocessingwitherror)
-    - [`onShippingRateSuccess`](#onshippingratesuccess)
-    - [`onShippingRateFail`](#onshippingratefail)
-    - [`onShippingRateSelectSuccess`](#onshippingrateselectsuccess)
-    - [`onShippingRateSelectFail`](#onshippingrateselectfail)
+-   [General Concepts](#general-concepts)
+    -   [Tracking flow through status](#tracking-flow-through-status)
+        -   [`CheckoutProvider` Exposed Statuses](#checkoutprovider-exposed-statuses)
+            -   [Special States:](#special-states)
+        -   [`ShippingProvider` Exposed Statuses](#shippingprovider-exposed-statuses)
+        -   [`PaymentMethodDataProvider` Exposed Statuses](#paymentmethoddataprovider-exposed-statuses)
+    -   [Emitting Events](#emitting-events)
+        -   [`onCheckoutValidationBeforeProcessing`](#oncheckoutvalidationbeforeprocessing)
+        -   [`onPaymentProcessing`](#onpaymentprocessing)
+        -   [`onCheckoutAfterProcessingWithSuccess`](#oncheckoutafterprocessingwithsuccess)
+        -   [`onCheckoutAfterProcessingWithError`](#oncheckoutafterprocessingwitherror)
+        -   [`onShippingRateSuccess`](#onshippingratesuccess)
+        -   [`onShippingRateFail`](#onshippingratefail)
+        -   [`onShippingRateSelectSuccess`](#onshippingrateselectsuccess)
+        -   [`onShippingRateSelectFail`](#onshippingrateselectfail)
```

Comment:
Could we maybe discard these changes? they add noise 😢

Reply:
They are just prettier changes, I'd prefer to leave them in if that's cool, otherwise next person to edit this file will have the same issue.